server/app.py

```diff
@@ -1146,22 +1138,39 @@ def stripehook():
 
     app.logger.info(f"Received event: id={event.id}, type={event.type}")
 
+    process_stripe_event(event)
+
+    return "Success", 200
+
+
+def process_stripe_event(event):
     # setting celery's delay on these keeps the stripe call from erroring out
     if event.type == "customer.source.updated":
         customer_source_updated.delay(event)
     if event.type == "payout.paid":
         payout_paid.delay(event)
     if event.type == "customer.subscription.created":
-        customer_subscription_created.delay(event)
+        subscription_id = event["data"]["object"]["id"]
```

Comment:
**suggestion: (non-blocking)** Could we store these as variables higher up? Or is it that sometimes an event won't have these keys in the event?

Something like:
```py
event_data = event["data"]["object"]
event_type = event.type # while we're at it
...
if event_type == "customer.subscription.created":
     subscription_id = event_data["id"]
```

Reply:
Great catch. Fixed.